lib/archethic/p2p/geo_patch.ex

```diff
@@ -29,68 +29,34 @@ defmodule ArchEthic.P2P.GeoPatch do
   end
 
   defp compute_patch(lat, lon) do
-    lat_sign = sign(lat)
-    lon_sign = sign(lon)
+    lat_pos = (lat + 90) / 11.25
+    lon_pos = (lon + 180) / 22.5
```

Comment:
@I-Archer-Zero 
Looks like added 90,180 to avoid negative sign but look at 

`
lat_pos 
`

Reply:
It must be
`lat_pos = (lat + 90) / 22.5`

Reply:
Hi !
Dividing lat by 22.5 will have a result between 0 and 8. But we are looking for a range between 0 and 16 to get all hexadecimal possibilities (0 to F).
Si we have to divide by 11.25 to get it.

I think on issue's example there is a mismatch. On the graph, latitude only have 8 parts, but on explication bellow it says that first 2 digits are on range of 0 to F so 16 parts...

Reply:
@I-Archer-Zero

Reply:
Hi @Neylix 
Good observation 

lat should be divided by 22.5 and not 11.25 because we need square patches (not rectangle)
This also means that the lat/22.5 will have a result between 0 to 8

Reply:
Hi ! Thanks for your return.
I didn't know it had to be a square, I'll update it as you requested. Also need to add 4 to result to get index from C to 3 as in your graph.
I also noticed that in issue you say that first digit is for longitude index, but I did latitude first, should I update it too ?
I'm actually out of my PC until sunday, I'll update it at this time.

Reply:
Hello ! 
I fixed calculation in lastest commits